src/fixit/tests/__init__.py

```diff
@@ -16,3 +16,7 @@
     globals(),
     collect_rules(enables=[QualifiedRule("fixit.rules")], disables=[]),
 )
+add_lint_rule_tests_to_module(
+    globals(),
+    collect_rules(enables=[QualifiedRule("fixit.rules.pyflakes")], disables=[]),
```

Comment:
This breaks the tests since the module doesn't (yet) exist. Let's go ahead and create an empty `__init__.py` there so that the tests can pass here.

Reply:
You can leave these lines in, and then add `src/fixit/rules/__init__.py`. That will provide the "fixit.rules.pyflakes" package that these lines are telling it to look for.

The following diffs in the stack accomplish the same thing, but it would be nice to have CI passing on all diffs in the stack, and the empty init will fill that role.